view/widgets/base_device_widget.py

```diff
@@ -122,7 +122,11 @@ def create_text_box(self, name, value):
 
         # TODO: better way to handle weird types that will crash QT?
         value_type = type(value)
-        textbox = QScrollableLineEdit(str(value))
+        if value_type == float:
+            # round floats to 2 decimal places
+            textbox = QScrollableLineEdit("{:.2f}".format(value))
+        else:
+            textbox = QScrollableLineEdit(str(value))
```

Comment:
Feels like maybe we shouldn't do it in base widget? Just seems like that's assuming a lot about every floating point input. Like for the stage, we need more that just 2 decimal places. Should probably handle in specific widget class.

Reply:
@micahwoodard agreed this could be done inside the device widgets themselves. I'll try to find the equivalent lines in each device class.

Reply:
@micahwoodard thanks I will take a first crack. What is the API call from the propert_widget to set this string format? Also, what are your thoughts on having this as a property of the deliminated property, in addition to the unit? Precision seems funny to stick in the property, its probably best set in the widget but just an idea.